sdk/documenttranslation/Azure.AI.DocumentTranslation/tests/Infrastructure/DocumentTranslationLiveTestBase.cs

```diff
@@ -53,7 +67,7 @@ public async Task<Uri> CreateSourceContainerAsync(List<TestDocument> documents)
                 await containerClient.UploadBlobAsync(documents[i].Name, stream);
             }
 
-            var expiresOn = DateTimeOffset.Now.AddHours(1);
+            var expiresOn = DateTimeOffset.UtcNow.AddHours(1);
```

Comment:
Per comment based on your PR description, this shouldn't be necessary. Whatever is serializing `expiresOn` should always write out the UTC-offset datetime or include the offset (i.e. either 'z' for UTC (or absent), or '+/-{offset}').

Reply:
Interesting. So this code is used to create a SAS token for the container. I am using the `Azure.Storage.Blobs` library.
The output  uses `Z` like, for example, this is what the SAS looked like before my change `https://mariaridoctrans29prim.blob.core.windows.net/source1990210606?sv=2020-04-08&se=2021-03-29T11:36:17Z&sr=c&sp=rl......`
After my change
`https://mariaridoctrans29prim.blob.core.windows.net/source1990210606?sv=2020-04-08&se=2021-03-29T18:38:32Z&sr=c&sp=rl...`

Both contain `Z` and differ only on +8h (rest numbers is how long it took me to run code again :P).

Are you then suggesting there might be a problem with the `GenerateSasUri` method?